### ISSUE TITLE
FSLA Fluctuation

### ISSUE THREAD

**Issue by clearwho** [ruohaoguo/avis#5]
Thanks for your great job.
After re-training several times, my FSLA is sometimes lower than the paper's. The FSLAn sub-metric also fluctuates a lot. Is this normal? 
Thanks.

**Comment by ruohaoguo** [comment]
Thanks for your attention.

This may be because the model overfits to certain cases, leading to worse predictions for some videos. If these videos contain more silent frames, it could cause significant fluctuations in FSLAn, as the total number of silent frames is relatively small (only 931, accounting for just 6.14%). The limited sample size of silent frames makes the metric more sensitive to variations.
